cypher/cypher-docs/src/docs/dev/advanced-query-tuning-example.asciidoc

```diff
@@ -18,7 +18,7 @@ Let's explain how to use these features with a more advanced query tuning exampl
 [NOTE]
 ====
 If you are upgrading an existing store to {neo4j-version-exact}, it may be necessary to drop and re-create existing indexes.
-For information on native index support and upgrade considerations regarding indexes, see <<operations-manual#index-configuration-btree,Operations Manual -> Indexes>>.
+For information on native index support and upgrade considerations regarding indexes, see <<operations-manual#index-configuration-range,Operations Manual -> Indexes>>.
```

Comment:
not sure this will be the correct link, but I guess we will see once @emmaholmbergohlsson/kernel have updated their parts